packages/rocketchat-lib/client/MessageAction.coffee

```diff
@@ -82,6 +82,7 @@ Meteor.startup ->
 			input = instance.find('.input-message')
 			Meteor.setTimeout ->
 				input.focus()
+				input.updateAutogrow()
 			, 200
```

Comment:
This timeout is annoying from user point of view. I am not sure why it is here so I did not remove it, though.